src/System.Private.ServiceModel/tests/Common/Scenarios/ServiceInterfaces.cs

```diff
@@ -105,6 +105,10 @@ public interface IWcfServiceGenerated
 
     [System.ServiceModel.OperationContractAttribute(Action = "http://tempuri.org/IWcfService/Echo", ReplyAction = "http://tempuri.org/IWcfService/EchoResponse")]
     System.Threading.Tasks.Task<string> EchoAsync(string message);
+
+    [System.ServiceModel.OperationContractAttribute(Action = "http://tempuri.org/IWcfService/EchoMessageParameter", ReplyAction = "http://tempuri.org/IWcfService/EchoMessageParameterResponse")]
+    [return: System.ServiceModel.MessageParameterAttribute(Name = "result")]
+    string EchoMessageParameter(string name);
```

Comment:
Consider updating this same service interface under the unit tests or consolidating them.  We have duplicate code in several places that should probably be kept in sync or combined.